src/components/Community/Events/index.tsx

```diff
@@ -112,11 +104,20 @@ const Events: React.FC<{ theme: ICommunitySectionTheme }> = ({ theme }) => {
         title={title}
       >
         <div className={sharedStyles.items}>
-          {eventsItems.map((event, key) => (
-            <div className={sharedStyles.item} key={key}>
-              {event && <Event {...event} theme={theme} key={event.url} />}
+          {events ? (
+            events.map((event, key) => (
+              <div className={sharedStyles.item} key={key}>
+                {event && <Event {...event} theme={theme} key={event.url} />}
+              </div>
+            ))
+          ) : (
+            <div className={styles.eventsPlaceholder}>
```

Comment:
one quick suggestion -from the design perspective - there are two ways we show text on the page - gray small and black large - let's use one of those for the text. I would start with grey small.

Reply:
Sounds good! I was wondering about something like that.

Reply:
Done, but you may want to see if I made the optimal implementation. I'm not sure where the right shared CSS to use would be, and don't seem to see any from context.